apps/web/pages/[user].tsx

```diff
@@ -28,40 +30,86 @@ interface EvtsToVerify {
 }
 
 export default function User(props: inferSSRProps<typeof getServerSideProps>) {
-  const { Theme } = useTheme(props.user.theme);
-  const { user, eventTypes } = props;
+  const { Theme } = useTheme(props.users[0].theme);
+  const { users } = props;
   const { t } = useLocale();
   const router = useRouter();
+
+  const eventTypes = props.users.length > 1 ? defaultEvents : props.eventTypes;
+  const groupEventTypes = props.users.some((user) => {
+    return !user.allowDynamicBooking;
+  }) ? (
+    <div className="space-y-6" data-testid="event-types">
+      <div className="overflow-hidden rounded-sm border dark:border-gray-900">
+        <div className="p-8 text-center text-gray-400 dark:text-white">
+          <h2 className="font-cal mb-2 text-3xl text-gray-600 dark:text-white">{" " + t("unavailable")}</h2>
+          <p className="mx-auto max-w-md">{t("user_dynamic_booking_disabled")}</p>
+        </div>
+      </div>
+    </div>
```

Comment:
Custom error when one of the users in the dynamic booking group have disallowed dynamic group booking.